plugins/autostart/src/lib.rs

```diff
@@ -234,10 +234,14 @@ impl Builder {
 /// Initializes the plugin.
 ///
 /// `args` - are passed to your app on startup.
-pub fn init<R: Runtime>(
+pub fn init<R: Runtime, I, S>(
     #[allow(unused)] macos_launcher: MacosLauncher,
-    args: Option<Vec<&'static str>>,
-) -> TauriPlugin<R> {
+    args: Option<I>,
```

Comment:
```suggestion
    // TODO: Remove this argument in v3
    args: Option<I>,
```

Reply:
the difference here is that macos_launcher must be set. well, with the current api design/intention at least. maybe it becomes more relevant with https://github.com/tauri-apps/plugins-workspace/issues/2720 but i wouldn't mind removing it from init either.

> But on the other hand, if we remove all the parameters, why not just use Builder::new().build()? 😂

That would be more in line with most plugins. They typically have either a builder or init, not both (with exceptions). In this case the builder was added at a later point.